scheduler/test/cook/test/kubernetes/controller.clj

```diff
@@ -83,7 +83,7 @@
                                                                                     :reason "Node preempted"
                                                                                     :pod-deleted? true
                                                                                     :pod-preempted? true})))
```

Comment:
Aren't we removing `:pod-preempted?` as a thing with this PR?

Reply:
for now, yes